subsys/config/Kconfig

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CONFIG_SYS_ENABLED
```

Comment:
this is going to be mega confusing, i.e. CONFIG_CONFIG_SYS_ENABLED, we should also drop the ENABLED part here, verbs are usually only used in hidden variables like HAS, SUPPORTED, ENABLED...

Maybe CONFIGURATION_SYSTEM? CFGSYS? or give it another neutral name

Reply:
will do